src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -330,14 +330,14 @@
   {:render
    (fn [{:keys [props]}]
      (let [sep [:span {} " " (icons/font-icon {:style {:fontSize "50%"}} :angle-right) " "]
-           crumbs (:crumbs props)]
+           crumbs (filter some? (:crumbs props))]
```

Comment:
why do you need this? will `(:crumbs props)` ever contain nil values? Or are you using `filter` just to ensure you have a sequence, if `(:crumbs props)` is itself nil?